api/db/migrations/20241028125952_add-indexes.js

```diff
@@ -0,0 +1,94 @@
+const up = async function (knex) {
+  // table 'certification-subscriptions'
```

Comment:
vu l'expressivité de l'api knex je pense qu'on peut omettre les commentaires

Reply:
pour exprimer plus directement les tables et index ajoutés, je propose
```suggestion
	const indexesToAdd = { 'certification-subscriptions': ['complementaryCertificationId', 'certificationCandidateId']},
	{'complementary-certifications': ['key']}
;

	for (const tableName of Object.keys(indexesToAdd)) {
		await knex.schema.table(tableName, function (table) {
			table.index(indexesToAdd[tableName]);
		});
	}
```

Reply:
J'ai préféré une syntaxe plus classique avec un object {table:"nomTable", index: ["column de l'index"]}, mais la PR est effectivement bien plus lisible avec une vraie symétrie par construction entre le up et le down